frontend/packages/dev-console/integration-tests/features/addFlow/create-from-catalog.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Create Application from Catalog file
+    As a user I want to create the application, component or service from Add Flow Catlog file
+
+    Background:
+        Given user is at developer perspective
+        And user has created namespace starts with "aut-addflow-catalog"
```

Comment:
Why `starts` is used over here

Reply:
For automation purpose, "starts with" is included, As we are creating the unique namespace